pandas/core/common.py

```diff
@@ -2646,7 +2646,7 @@ def _astype_nansafe(arr, dtype, copy=True):
     return arr.view(dtype)
 
 
-def _clean_fill_method(method):
+def _clean_fill_method(method, allow_nearest=False):
```

Comment:
this seems kind of hacky, why don't we just always allow nearest (I know its not quite supported by fillna). Or will you just fix this then?